cli/internal/scope/scope.go

```diff
@@ -165,7 +165,8 @@ func calculateInference(repoRoot turbopath.AbsoluteSystemPath, rawPkgInferenceDi
 		if err != nil {
 			return nil, err
 		}
-		if inferredPathIsBelow {
+		// We skip over the root package as the inferred path will always be below it
```

Comment:
I don't know much about this feature, but is this true? If you run `turbo run` from the root of the repo, will inference be skipped altogether?

Reply:
`turbo run` from repo root will cause `rawPkgInferenceDir` to be `""` which triggers a return at the beginning of this function